onnxruntime/core/providers/openvino/backends/basic_backend.cc

```diff
@@ -545,14 +545,16 @@ void BasicBackend::Infer(OrtKernelContext* ctx) {
       std::cout << "Inference successful" << std::endl;
     }
 
+    OVInferRequestPtr infer_request_ = infer_request;
```

Comment:
@preetha-intel  Amended your request in the parent branch. Please checkout commit id : 82b872fbf2ae823cb1349d55c635cbe57a00e00d
I want to avoid any more PRs for the same, please cherry-pick for your upstream